assessment.md

```diff
@@ -13,81 +13,512 @@
+  * I = An attempt was made to add a status badge for Travis CI builds but does
+  not work or show up
+  * A = A status badge for Travis CI builds is in the README but is not in an
+  appropriate location
+  * G = A status badge for Travis CI builds is in the README and is in an
+  appropriate location but not in relation to any other status badges
+  * E = A status badge for Travis CI builds is in the README and is in an
+  appropriate location in relation to any other status badges
 * Add README badges that show project characteristics (e.g., coverage and
   language)
+  * N = No status badges were added besides one for the build status of Travis
+  CI
+  * I = Only some of the relevant status badges were added besides one for the
+  build status of Travis CI
+  * A = All relevant status badges were added but are not in appropriate
+  locations
+  * G = All relevant status badges were added and are in an appropriate location
+  but not in relation to each other
+  * E = All relevant status badges were added and are in an appropriate location
+  in relation to each other and a potentially present status badge for Travis CI
+  builds
 
 ### Foundations of Software Engineering
 
 * Requirements engineering
+  * N = No effort to document, define, or maintain requirements present
+  * I = Brief or unspecific documentation, definition, and maintenance given
+  * A = Explains what was done with enough detail to convey the most important information
+  * G = Documentation and definition are well written and gives other team members
+  an informed idea of the process
+  * E = The software engineer displayed highly detailed, elaborate, and complete
+  documentation, definition, and maintenance in the given design process
 * Software architecture
+  * N = There is no concrete structure and the master branch is broken
+  * I = There are small amounts of structure to the code where it may or may not
+  run correctly, and the master branch is broken
+  * A = There is a blueprint for the software system that lacks completion, and the
+  master branch is not broken
+  * G = There is a completed blueprint for the software system and a working master
+  branch
+  * E = The software system contains high-level structures that contain software
+  elements in a complex and complete blueprint of the software system, and the
+  master branch has no errors and contains completed code
 * Software design
+  * N = No problem solving steps taken
+  * I = Initial steps are taken to conceptualize, frame, and implement systems
+  * A = Some software design is visible to the point where a software solution looks
+  to be developed soon
+  * G = A software solution is created that includes both an algorithm design and
+  a high-level architecture design
+  * E = All goals and expectations of the design are met and compatibility, reliability,
+  reusability, and scalability are included    while a software solution is ultimately
```

Comment:
There is unnecessary spacing on line 303.

Reply:
I will handle this one